e2e/src/parent.ts

```diff
@@ -0,0 +1,202 @@
+/**
+ * This is a testing / demo scaffold that simulates the Source web application
+ * loading an iframe plugin. It's super hacky and hard-coded, but it is just enough
+ * to get an iframe plugin bootstrapped and communicating over the SourceBridge API.
+ */
+
+const APPLICATION_ID = 'app_123' // matches the demo server's dummy config
+
+function generateRequestId(): string {
```

Comment:
[future/nit] I've noticed we duplicate this a few times, we ought to put this into a util library somewhere.

Reply:
I deliberately didn't pull this in from the Bridge SDK since the parent page won't be using it in real life.

The bridge SDK has a version of this function, and we want to avoid adding dependencies to that so it makes sense to keep it in there. On our backend, we have an ID generation function that we use all over the place, backed by the nanoid library.